package.json

```diff
@@ -86,6 +89,14 @@
     "typescript": "5.4.5",
     "url-loader": "4.1.1"
   },
+  "lint-staged": {
```

Comment:
could we move the configuration to a `.lintstagedrc.json` file in the project root like out other configs?